obs_status.yml

```diff
@@ -0,0 +1,2147 @@
+  1108353064: 9
+  1117787424: 9
+  1130649544: 9
+  1139171624: 9
+  1178221128: 9
+  1197635184: 9
+mags:
+- agasc_id: 5115672
+  mag_aca: 11.422
+  mag_aca_err: 0.1
+- agasc_id: 33690776
+  mag_aca: 9.984
+  mag_aca_err: 0.1
+- agasc_id: 34996472
+  mag_aca: 10.562
+  mag_aca_err: 0.1
+- agasc_id: 38535536
+  mag_aca: 10.719
+  mag_aca_err: 0.1
+- agasc_id: 40113272
+  mag_aca: 11.781
+  mag_aca_err: 0.1
+- agasc_id: 45616472
+  mag_aca: 11.422
+  mag_aca_err: 0.1
+- agasc_id: 47846640
+  mag_aca: 10.891
+  mag_aca_err: 0.1
+- agasc_id: 48892720
+  mag_aca: 11.484
+  mag_aca_err: 0.1
+- agasc_id: 49818760
+  mag_aca: 10.859
+  mag_aca_err: 0.1
+- agasc_id: 62922104
+  mag_aca: 10.891
+  mag_aca_err: 0.1
+- agasc_id: 68956456
+  mag_aca: 11.328
+  mag_aca_err: 0.1
+- agasc_id: 82972168
+  mag_aca: 11.2
+  mag_aca_err: 0.1
+- agasc_id: 92812824
+  mag_aca: 11.141
+  mag_aca_err: 0.1
+- agasc_id: 107881800
+  mag_aca: 10.969
+  mag_aca_err: 0.1
+- agasc_id: 109061768
+  mag_aca: 11.621  # need to check, not from starcheck report
+  mag_aca_err: 0.1
+- agasc_id: 110493968
+  mag_aca: 10.3
+  mag_aca_err: 0.1
+- agasc_id: 111028544
+  mag_aca: 10.516
+  mag_aca_err: 0.1
+- agasc_id: 111807640
+  mag_aca: 12.266
+  mag_aca_err: 0.1
+- agasc_id: 118359864
+  mag_aca: 11.25
+  mag_aca_err: 0.1
+- agasc_id: 118498608
+  mag_aca: 12.7
+  mag_aca_err: 0.1
+- agasc_id: 120457376
+  mag_aca: 11.547
+  mag_aca_err: 0.1
+- agasc_id: 144714600
+  mag_aca: 10.097  # need to check, not from starcheck report
+  mag_aca_err: 0.1
+- agasc_id: 145113736
+  mag_aca: 11.234
+  mag_aca_err: 0.1
+- agasc_id: 152831696
+  mag_aca: 10.656
+  mag_aca_err: 0.1
+- agasc_id: 161357600
+  mag_aca: 11.094
+  mag_aca_err: 0.1
+- agasc_id: 166993856
+  mag_aca: 11.203
+  mag_aca_err: 0.1
+- agasc_id: 185871616
+  mag_aca: 11.391
+  mag_aca_err: 0.1
+- agasc_id: 186125624
+  mag_aca: 11.6
+  mag_aca_err: 0.2
+- agasc_id: 187829568
+  mag_aca: 11.2
+  mag_aca_err: 0.1
+- agasc_id: 188482256
+  mag_aca: 8.266
+  mag_aca_err: 0.1
+- agasc_id: 188501784
+  mag_aca: 10.453
+  mag_aca_err: 0.1
+- agasc_id: 188623712
+  mag_aca: 11.297
+  mag_aca_err: 0.1
+- agasc_id: 188750920
+  mag_aca: 11.469
+  mag_aca_err: 0.1
+- agasc_id: 195167088
+  mag_aca: 12.7
+  mag_aca_err: 0.1
+- agasc_id: 225576592
+  mag_aca: 11.297
+  mag_aca_err: 0.1
+- agasc_id: 239080480
+  mag_aca: 10.2
+  mag_aca_err: 0.1
+- agasc_id: 257953336
+  mag_aca: 11.594
+  mag_aca_err: 0.1
+- agasc_id: 264115840
+  mag_aca: 12.844
+  mag_aca_err: 0.1
+- agasc_id: 288753656
+  mag_aca: 11.359
+  mag_aca_err: 0.1
+- agasc_id: 306200176
+  mag_aca: 10.953
+  mag_aca_err: 0.1
+- agasc_id: 311824376
+  mag_aca: 11.25
+  mag_aca_err: 0.1
+- agasc_id: 327945264
+  mag_aca: 11.25
+  mag_aca_err: 0.1
+- agasc_id: 328077952
+  mag_aca: 11.016
+  mag_aca_err: 0.1
+- agasc_id: 330967424
+  mag_aca: 10.947  # need to check, not from starcheck report
+  mag_aca_err: 0.1
+- agasc_id: 350392600
+  mag_aca: 5.0  # need to check
```

Comment:
This one is also probably in a the bad star table (it was in the proseco bad star list) and there we said it was like a 3.6 mag star.  I am concerned that the agasc supplement report doesn't really give clues about it being too bright.

Reply:
Any star that is in the 'bad' table already should not be here.

All the following are in the obs_status file, but they are already in the bad table, so I will remove them from this file:
- 33690776
- 34996472
- 111028544
- 185871616
- 350392600
- 444743456
- 656409216
- 1117787424
- 1130649544
- 1197635184

(there will be a commit for this)

Reply:
there is still the question whether we would like the magnitudes for these stars in the mags table, but considering that the magnitude estimate is usually some lower/upper limit, and the star is already marked bad, I do not see the value.